Formula/n/newnode-helper.rb

```diff
@@ -0,0 +1,39 @@
+class NewnodeHelper < Formula
+  desc "Web proxy that uses a distributed p2p network to circumvent censorship"
+  homepage "https://www.newnode.com/newnode-vpn"
+  url "https://github.com/clostra/newnode.git",
+    tag: "2.1.4", revision: "c42a04ded55cfdff857878f4d9234950e530bb00"
+  license ""
+
+  depends_on xcode: ["9.3", :build]
+  depends_on "automake"
+  depends_on "coreutils"
+  depends_on "libtool"
+  depends_on "mbedtls@2"
+  depends_on "wget"
+
+  def install
+    system "git", "clone", "--recurse-submodules", "https://github.com/clostra/newnode.git"
```

Comment:
this should be covered with the tag checkout
```suggestion
```

Reply:
done